packages/mask/src/plugins/NextID/components/BindingItem.tsx

```diff
@@ -53,8 +53,10 @@ const useStyles = makeStyles()((theme) => ({
         marginLeft: theme.spacing(1),
     },
 }))
+
+type Platform = 'twitter' | 'nextid' | 'keybase' | 'ethereum' | 'github'
 interface Item {
-    platform: NextIDPlatform
+    platform: Platform
```

Comment:
Why not use `NextIDPlatform `?

Reply:
cause NextIDPlatform is a enum type，and platform is a string type